Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -128,6 +128,31 @@ public extension Node where Context: HTMLLinkableContext {
     }
 }
 
+public extension Attribute where Context == HTML.LinkContext {
+    /// Assign a hreflang string to this element.
+    /// - parameter language: The language to assign.
+    /// - parameter country: The country of the language to assign.
+    static func hreflang(_ language: Language, _ country: Country? = nil) -> Attribute {
+        Attribute(name: "hreflang", value: country != nil ? "\(language.rawValue)-\(country?.rawValue ?? "")": language.rawValue)
+    }
+}
+
+public extension Attribute where Context == HTML.LinkContext {
+    /// Assign an icon sizes string to this element.
+    /// - parameter sizes: The icon sizes of the link to assign.
+    static func sizes(_ sizes: String) -> Attribute {
+        Attribute(name: "sizes", value: sizes)
+    }
+}
+
+public extension Attribute where Context == HTML.LinkContext {
```

Comment:
This extension can be merged with the above two ones. No need to create a new extension for each new API.

Reply:
done.